var/spack/repos/builtin/packages/mimalloc/package.py

```diff
@@ -21,3 +21,6 @@ class Mimalloc(CMakePackage):
     version('1.7.6', sha256='d74f86ada2329016068bc5a243268f1f555edd620b6a7d6ce89295e7d6cf18da')
 
     depends_on('cmake@3.0:', type='build')
+
+    def cmake_args(self):
+        return [self.define('MI_INSTALL_TOPLEVEL', True)]
```

Comment:
This seems reasonable, but the option is marked deprecated: https://github.com/microsoft/mimalloc/blob/f2712f4a8f038a7fb4df2790f4c3b7e3ed9e219b/CMakeLists.txt#L28. Is this a must and is something going to break when the option is removed?

Reply:
Hm, mimalloc is annoying :( they have this `include/mimalloc-<major>.<minor>/mimalloc-new-delete.h` structure, but mold doesn't know the mimalloc version, and there are no pkg-config files. I think this is the easiest solution, and I think it's OK for hpx/pika since they use cmake config files.

Reply:
Yeah, I think pika (CMake, really) won't care if this is enabled or not. I was just worrying about what happens when that option is removed. Then again we can deal with that the day it happens as well, so please don't consider this comment a blocker.

Reply:
Adding pkg-config support to mimalloc & using that in mold is probably best, but I don't want to waste time on it ;p

Reply:
Hm, sorry for all the noise. As it is mold links to mimalloc dynamically and it should link statically :|. I'll just leave mimalloc a vendored dependency until mimalloc has pkg-config support.